test/civ2.test.js

```diff
@@ -0,0 +1,699 @@
+            [
+              { method: 'reply', args: ['Deleting branch toto/pipo...'] },
+              { args: ['SPY:deleteBranches'], method: 'reply' }
+            ]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error('delete branch toto/pipo', [
+            { command: 'deleteBranches', expectedArgs: ['toto/pipo', 'John Doe'] }
+          ]));
+      });
+
+      describe('Link update', () => {
+        it('should update PR description', async () =>
+          expectCiv2CommandCall(
+            'update links toto/pipo',
+            [{ command: 'updatePRsDescriptions', expectedArgs: ['toto/pipo', 'John Doe'] }],
+            [
+              { method: 'reply', args: ['Updating links for branch toto/pipo...'] },
+              { args: ['SPY:updatePRsDescriptions'], method: 'reply' }
+            ]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error('update links toto/pipo', [
+            { command: 'updatePRsDescriptions', expectedArgs: ['toto/pipo', 'John Doe'] }
+          ]));
+      });
+
+      describe('Instance update', () => {
+        it('should trigger instance update with default args', async () =>
+          expectCiv2CommandCall(
+            'update instance pouet',
+            [{ command: 'updateInstance', expectedArgs: ['pouet', undefined, undefined] }],
+            [
+              {
+                method: 'reply',
+                args: ['Instance <pouet|pouet> on "undefined" is updating to version "SPY:updateInstance"']
+              }
+            ]
+          ));
+
+        it('should trigger instance update with specific env', async () =>
+          expectCiv2CommandCall(
+            'update instance pouet on shiny environment',
+            [{ command: 'updateInstance', expectedArgs: ['pouet', 'shiny', undefined] }],
+            [
+              {
+                method: 'reply',
+                args: ['Instance <pouet|pouet> on "shiny" is updating to version "SPY:updateInstance"']
+              }
+            ]
+          ));
+
+        it('should trigger instance update with specific env and version', async () =>
+          expectCiv2CommandCall(
+            'update instance pouet on shiny environment to version ultimate',
+            [{ command: 'updateInstance', expectedArgs: ['pouet', 'shiny', 'ultimate'] }],
+            [
+              {
+                method: 'reply',
+                args: ['Instance <pouet|pouet> on "shiny" is updating to version "SPY:updateInstance"']
+              }
+            ]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error('update instance pouet on shiny environment to version ultimate', [
+            { command: 'updateInstance', expectedArgs: ['pouet', 'shiny', 'ultimate'] }
+          ]));
+      });
+
+      // HELPERS
+      // -----------------------------------------------------------------------------
```

Comment:
If you use a section-comment to separate helpers, you should do the same at the beginning of a `describe`. Else, it looks that those helpers are linked to the describe just below, when browsing the file. Sections are hierarchical :) Or just don't use a separator for helpers. In any case, the emphasis must be on `describe`, not on helpers.